x-pack/plugins/security_solution/server/endpoint/services/artifacts/unified_manifest_client.ts

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type {
+  SavedObjectsBulkDeleteResponse,
+  SavedObjectsBulkResponse,
+  SavedObjectsBulkUpdateResponse,
+  SavedObjectsClientContract,
+  SavedObjectsFindResponse,
+} from '@kbn/core-saved-objects-api-server';
+import { createSoFindIterable } from '../../utils/create_so_find_iterable';
+import { mapUnifiedManifestSavedObjectToUnifiedManifest } from './utils';
+import type {
+  InternalUnifiedManifestCreateSchema,
+  InternalUnifiedManifestBaseSchema,
+  InternalUnifiedManifestSchema,
+  InternalUnifiedManifestUpdateSchema,
+} from '../../schemas';
+import { ManifestConstants } from '../../lib/artifacts';
+
+interface FetchAllUnifiedManifestsOptions {
+  perPage?: number;
+  kuery?: string;
+  sortField?: string;
+  sortOrder?: 'desc' | 'asc';
+  fields?: string[];
+}
+
+export const UNIFIED_MANIFEST_ALL_NAMESPACES = '*';
```

Comment:
Setting the initial namespace on create and running other operations using global namespace. This gives us an experience of `agnostic` namespace type while keeping the door open for future space awareness implementation.